user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+and even export all the data in the grid as a .csv file.
+
+Get to a Grid
+-------------
+
+There are several ways to get to a grid.
+
+- Select the relevant point in the :ref:`menu bar <user-guide-navigation-menu>`: to get to the *"All Contacts"* grid, 
+  select the *"Contacts"* menu item, to get to the *"All Reports"* grid, click the *"Manage Custom Reports"* menu item, 
+  etc. 
+  
+- Find it in the :ref:`Shortcuts <user-guide-getting-started-shortcuts>`.
+
+- Click the grid link on the :ref:`View page <user-guide-ui-components-view-pages>` 
+  or :ref:`Create/Edit form <user-guide-ui-components-create-pages>` of a record. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_from_view.png
+
+|
+
+.. hint::
+
+    If you have reached a View page or Create/Edit form from a grid, and now click  link to get back to the grid, this 
+    grid will be look in the same as it did when you left it for the View page (the same filters and order applied).    
+
+    
+
+Adjust a Grid
+-------------
+
+.. _user-guide-ui-components-grids-change-table:
+
+Change the Table
+^^^^^^^^^^^^^^^^
+
+Actually, you can change the set of details available in the grid (if permitted), and the order of columns.
+To do so, go to the right corner of the grid, and click the settings icon |IcSettings|.
+
+To show/hide a column on the grid, check/uncheck it in the *"Table Settings"*.(The list of columns is defined 
+by the system administrator with the  *Show On Grid* setting of the entity fields).
+
+To move columns of the grid, move them in the form: click on the "Sort" column and hold the mouse button - drag the 
+column to change their order:
+
+|
+
+.. image:: ./img/data_management/grid/grid_table_settings.png
+
+|
+
+.. _user-guide-ui-components-grid-filters:
+
+Apply Filters
+^^^^^^^^^^^^^
+
+You can apply filters to choose specific items to be shown in the grid. 
+
+Of course, if there are a lot of contacts, they all won't fit to one page. In order to find the required contact in the 
+grid, you can use the grid filters. 
+
+Some filters are available by default, but the set of filters can be modified with check-boxes under the 
+*"Manage Filters"* menu on the left. (The list of filters is defined 
+by the system administrator with the *Show Grid Filter* setting of the entity fields).
+
+|
+
+.. image:: ./img/data_management/grid/grid_filters.png
+
+|
+
+The list of properties that can be used as filters is defined by the system administrator with *"Show Grid Filter"* 
+setting of the entity fields. 
+
+
+Click the arrow on a filter to choose required values. Available value setting depend on the field type. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_filters_define.png
+
+|
+
+If more than one filter is active, only the records that meet requirements of ALL the filters are displayed. For 
+example, we have filtered the *"All Contacts"* grid so that it would only show Jameses Browns from California.
```

Comment:
Edit: If more than one filter is active, only records that meet the requirements of ALL the filters will be displayed. For example, we have filtered the *"All Contacts"* grid so that it only shows James Browns from California: